app/views/accounts/actions/_generate_backup_codes.html.erb

```diff
@@ -1,6 +1,6 @@
 <%= render ButtonComponent.new(
       action: ->(**tag_options, &block) do
-        link_to(backup_code_create_path, **tag_options, &block)
+        link_to(backup_code_setup_path, **tag_options, &block)
```

Comment:
This confused me that we might have had an existing link, but the route being removed associated with the named `backup_code_create` is `PATCH` method, but this link is `GET`, so it directs to the still-existing `backup_code_setup#index`. This link probably should have never used the `backup_code_create_path` helper.